scripts/assets/test_file_exemptions.textproto

```diff
@@ -555,6 +556,7 @@ exempted_file_path: "domain/src/main/java/org/oppia/android/domain/topic/Revisio
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/util/FloatExtensions.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/util/FractionExtensions.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/util/JsonAssetRetriever.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/workmanager/WorkManagerConfigurationModule.kt"
```

Comment:
This is probably in alphabetical order so move this below `InteractionObjectTestBuilder`

Reply:
InteractionObjectTestBuilder has this import statement
**"domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt"**

whereas WorkManagerConfigurationModule has this 
**"domain/src/main/java/org/oppia/android/domain/workmanager/WorkManagerConfigurationModule.kt"**

Reply:
Oh ah I see, thanks for clarifying!